ur_robot_driver/test/trajectory_test.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+import sys
+import time
+import unittest
+
+import rospy
+import actionlib
+from control_msgs.msg import FollowJointTrajectoryAction, FollowJointTrajectoryGoal, FollowJointTrajectoryResult
+from ur_dashboard_msgs.msg import SetModeAction, SetModeGoal, RobotMode
+from std_srvs.srv import Trigger, TriggerRequest
+from trajectory_msgs.msg import JointTrajectoryPoint
+
+PKG = 'ur_rtde_driver'
+NAME = 'trajectory_test'
+
+
+class TrajectoryTest(unittest.TestCase):
+    def __init__(self, *args):
+        super(TrajectoryTest, self).__init__(*args)
+        rospy.init_node('trajectory_testing_client')
+        self.client = actionlib.SimpleActionClient(
+            '/scaled_pos_traj_controller/follow_joint_trajectory', FollowJointTrajectoryAction)
```

Comment:
Instead of hard-coding the topic name, it might be nicer to only spec `follow_joint_trajectory` here and then remap in the `.test` file.